src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

```diff
@@ -0,0 +1,246 @@
+
+The APM summary page is an overview of the important metrics for your service. It's the first place you will go to investigate the source of a service degradation. On the APM summary page you can check the health of your service, understand any problems within a larger context, and take the necessary steps to resolve the issue. 
+
+<img
+    title="apm summary page"
+    alt="A screenshot depicting the APM summary page."
+    src={apmApmCompleteSummaryPage}
+/>
+
+This doc walks you through a _**fictional**_ example of how to use New Relic's APM tools to determine the root cause of service degradation. We're going to explore a situation where **you work for a fictional e-commerce company that sells shoes**. You are an engineering manager the checkout team.  Customers have been complaining they have to wait too long to checkout. 
+
+To investigate, open APM and select your `Checkout-service`.
+
+
+<Steps>
+<Step>
+
+## First, review these key metrics
+Is there a problem with your `Checkout-service`? Which systems are affected and how?
+
+<CollapserGroup>
+  <Collapser
+    id="apdex"
+    title="Apdex"
+  >
+
+<img
+    title="apdex in APM"
+    alt="A screenshot depicting a sample Apdex score in APM"
+    src={apmApdexinApm}
+/>
+The first place to look when investigating a breakdown in service is your Apdex score. Your Apdex score monitors overall customer satisfaction with your application. Your score looks for a combination of performance, like response time or throughput, and error rates. 
+
+Apdex is an industry standard that measures your users' satisfaction with the response time of your web application/service. It's represented as a score from 0-1. The closer your score is to 1, the better your app is performing. The default value for a satisfactory experience is 0.5 seconds, but you can set a different target under Settings.
+
+Your Apdex score is typically divided in the following colors:
+
+* ** &lt; 0.5 - Gray:** Your application's performance is beyond critical and needs immediate action.
+* **0.5-0.7 - Red:** There are critical performance issues in your application and it needs immediate action. 
+* **0.7-0.85 -Orange:** Your application is trending in a negative direction and needs further investigation. 
+* **0.85-0.95 - Blue:** This is the ideal Apdex range. This score means that you have fine-tuned your Your Apdex T perfectly for your application and your performance is healthy. 
+* **&gt; 0.95 - Blue:** If your Apdex score is this high it means that your Apdex T might be set a little too high and you're not getting an accurate read on your application's performance.. You should consider reducing your Apdex T.
```

Comment:
Repeated use of high in a sentence, I'd suggest "if you're apdex score is in this range, it means that..."

Reply:
Double period before the last sentence